src/main/java/com/yunhalee/walkerholic/activity/domain/Activity.java

```diff
@@ -43,7 +43,14 @@ public Activity(String name, Integer score, String description) {
         this.description = description;
     }
 
+    // 비지니스 로직
+    public Activity updateActivity( Activity requestActivity) {
```

Comment:
```suggestion
    public Activity updateActivity(Activity requestActivity) {
```
google java style guide가 익숙치 않다면, IntelliJ 또는 Eclipse의 formatting 기능을 활용해보세요.
> https://google.github.io/styleguide/javaguide.html

IntelliJ 단축키는  Ctrl+Alt+L(윈도우), Cmd+Alt+L(맥)
Eclipse 단축키는 Ctrl+Shift+F(윈도우), Cmd+Shift+F(맥)

Reply:
확인하여 수정하였습니다!